matrix.yml

```diff
@@ -44,6 +46,7 @@ include:
 - os: "ubuntu:20.04"
   images:
   - features: [*oneapi_2022, *cuda_curr_max, *python, *lit]
+  - features: [*gcc_6, *cuda_curr_max, *python, *lit]
```

Comment:
gcc-6 has no installable packages for 20.04
```suggestion
```

Reply:
Switched to making the gcc6 + cuda12.1 container based on 18.04.